File: Lession 1/Factorial.py
def Factorial(n):
    if n == 0:
        return 1
    return n * Factorial(n - 1)

def main():
    n = int(input("Enter a number: "))  # Chuyển đổi input thành số nguyên
    print(Factorial(n))

if __name__ == "__main__":
    main()

File: Buoi1/Factorial.cpp
#include <iostream>
using namespace std;
int Factorial(int n)
{
    if (n == 0)
    {
        return 1;
    }
    return n * Factorial(n - 1);
}
int main()
{
    int n;
    cout << "Enter n: ";
    cin >> n;
    cout << "Factorial of " << n << " is: " << Factorial(n);
    return 0;
}